ipcrm, ipcs: provided by cygwin.
readprofile: requires /proc/profile (Linux-specific)

--- origsrc/util-linux-ng-2.15.1/sys-utils/Makefile.am	2009-06-28 20:42:52.838444300 -0500
+++ src/util-linux-ng-2.15.1/sys-utils/Makefile.am	2009-06-28 20:40:13.057252900 -0500
@@ -2,20 +2,19 @@ include $(top_srcdir)/config/include-Mak
 
 bin_PROGRAMS =
 sbin_PROGRAMS =
-usrbinexec_PROGRAMS = flock ipcrm ipcs ipcmk renice setsid
-usrsbinexec_PROGRAMS = readprofile
+usrbinexec_PROGRAMS = flock ipcmk renice setsid
+usrsbinexec_PROGRAMS =
 
-dist_man_MANS = flock.1 ipcrm.1 ipcs.1 ipcmk.1 renice.1 setsid.1 \
-		readprofile.1
+dist_man_MANS = flock.1 ipcmk.1 renice.1 setsid.1
 
 if LINUX
 bin_PROGRAMS += dmesg
 sbin_PROGRAMS += ctrlaltdel
-usrbinexec_PROGRAMS += cytune setarch lscpu
-usrsbinexec_PROGRAMS += ldattach tunelp rtcwake
+usrbinexec_PROGRAMS += cytune ipcrm ipcs setarch lscpu
+usrsbinexec_PROGRAMS += ldattach tunelp readprofile rtcwake
 
-dist_man_MANS += dmesg.1 ctrlaltdel.8 cytune.8 setarch.8 \
-		ldattach.8 lscpu.1 tunelp.8 rtcwake.8
+dist_man_MANS += dmesg.1 ipcrm.1 ipcs.1 ctrlaltdel.8 cytune.8 setarch.8 \
+		ldattach.8 lscpu.1 tunelp.8 readprofile.1 rtcwake.8
 endif
 
 cytune_SOURCES = cytune.c cyclades.h
@@ -40,6 +39,7 @@ RDEV_LINKS += ramsize vidmode rootflags
 dist_man_MANS += rdev.8
 endif
 
+if LINUX
 SETARCH_LINKS = linux32 linux64
 
 if ARCH_S390
@@ -66,6 +66,7 @@ endif
 if ARCH_HPPA
 SETARCH_LINKS += parisc parisc64 parisc32
 endif
+endif
 
 RDEV_MAN_LINKS = $(RDEV_LINKS:=.8)
 SETARCH_MAN_LINKS = $(SETARCH_LINKS:=.8)
